sdk/core/core-http/src/policies/tracingPolicy.ts

```diff
@@ -78,7 +79,7 @@ export class TracingPolicy extends BaseRequestPolicy {
       // set headers
       const spanContext = span.spanContext();
       const traceParentHeader = getTraceParentHeader(spanContext);
-      if (traceParentHeader) {
+      if (traceParentHeader && isSpanContextValid(spanContext)) {
```

Comment:
We should not inject invalid tracerheader values (like all 0s, or not enough 0s, etc). This uses the helper function from opentelemetry/api that validates that the spanContext is valid.